sihl_users/lib/database.ml

```diff
@@ -0,0 +1,19 @@
+module MariaDb = struct
```

Comment:
I think we should find a way of creating separate opam package that contains MariaDB or PostgreSQL implementation of the users repository. Then that would require the app to have matching Db implementation linked.

Reply:
We can address this later.

Reply:
Yes, but for now we can ship it just as default implementation.